src/pages/verify.tsx

```diff
@@ -1,10 +1,37 @@
-import React from "react";
+import React, { FunctionComponent, useContext } from "react";
 import { Helmet } from "react-helmet";
 import { DropZoneSectionContainer } from "../components/VerifyPageContent/DropZoneSection";
 import { Page } from "../components/Layout/Page";
 import { useProviderContext } from "../common/contexts/provider";
 import { Link } from "react-router-dom";
-import { ErrorPage } from "@govtechsg/tradetrust-ui-components";
+import {
+  Button,
+  ErrorPage,
+  Overlay,
+  OverlayContext,
+  OverlayContextProvider,
+  Textual,
+} from "@govtechsg/tradetrust-ui-components";
+import { NetworkSelect } from "../components/Layout/NetworkSelect";
+export interface OverlayProps {
+  buttonText: string;
+  className: string;
+  style: Record<string, string | number>;
+  children: React.ReactNode;
+}
+
+const QuestionOverlay: FunctionComponent<OverlayProps> = ({ buttonText, className, style, children }) => {
+  const { showOverlay } = useContext(OverlayContext);
+
+  return (
+    <>
+      <Overlay />
+      <Button className={className} style={style} onClick={() => showOverlay(children)}>
+        {buttonText}
+      </Button>
+    </>
+  );
+};
```

Comment:
can find a place to put this, like in component/UI/Overlay/QuestionOverlay, then import inside here?

Reply:
makes sense to do so, thanks